react-effects/my-app/src/List.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable no-unused-vars -- REMOVE ME */
+import { useEffect, useState } from 'react';
+import readItems from './read';
+
+export default function List() {
+  const [isLoading, setIsLoading] = useState(undefined);
+  const [items, setItems] = useState([]);
+  const [error, setError] = useState();
+
+  useEffect(() => {
+    const readMe = async () => {
+      setIsLoading(true)
+      const data = await readItems();
+      setItems(data);
+      setIsLoading(false);
+    }
+    if (isLoading) {
```

Comment:
It looks like this will cause your effect to run twice, once to set isLoading to true, and once to make the read. Not a big problem, but you could avoid it by checking `isLoading === undefined`.